### PR TITLE
Review feedback

### LINKED ISSUE #4: Why not uv?
Some users might be in an organisation with a very restricted environment. `pixi` is just harder to install than `uv`. `uv` can be installed with `pip`. If you are a windows user you will need to install pixi via a windows installer which might not be allowed or possible. I think there are other ways. But the pixi web site describes so many different ways of installing that it appears complicated. There is no simple, single way to install no matter what.

Furthermore `uv` is a very well known and widely adapted tool already while pixi appears more niche. For example [Ray Bell writes](https://sites.google.com/view/raybellwaves/blog/prefix-dev-tools) he uses pixi for those 10% of projects that are very hard to install and `uv` for the rest.

Please consider migrating from `pixi` to `uv` to ease adaption.

### LINKED ISSUE #6: Replace isort with ruff
Ruff can also replace isort. Please replace it in the .pre-commit-config.yaml.

### LINKED ISSUE #9: mkdocs: command not found
I cannot run `docs-build`.

```bash
$ pixi run docs-build
✨ Pixi task (docs-build in default): mkdocs build
mkdocs: command not found
```

### LINKED ISSUE #10: auto docstring does not work locally
To get autodoc string working locally I need to remove `src.`.

image

But if I do it [does not work in Github Actions](https://github.com/awesome-panel/panel-copy-paste/actions/runs/11995963030/job/33439858341).

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
This project is awesome!

Here is my feedback based on using it to create https://github.com/awesome-panel/panel-copy-paste.

### CODE REVIEW

File: template/pyproject.toml.jinja [hunk 1]
```diff
@@ -93,6 +93,8 @@ fix = true
 
 [tool.ruff.lint]
 ignore = [
+    "D203", # one-blank-line-before-class and `no-blank-line-before-class` (D211) are incompatible.
```
Comment (MarcSkovMadsen) [inline template/pyproject.toml.jinja:96]: These ignore selections requires update  of some of your docstrings. I don't mind using other rule. But we should align with Param. See https://github.com/holoviz/param/pull/982.

File: template/pixi.toml.jinja [hunk 2]
```diff
@@ -84,6 +82,9 @@ depends_on = ["_install-ui"]
 mkdocs = "*"
 mkdocs-material = "*"
 mkdocstrings-python = "*"
+[feature.docs.tasks]
```
Comment (ahuang11) [inline template/pixi.toml.jinja:86]:
```suggestion

mkdocstrings-python = "*"
[feature.docs.tasks]
```

File: template/pyproject.toml.jinja [hunk 3]
```diff
@@ -119,6 +122,9 @@ select = [
     "UP036",
 ]
 
+[tool.ruff.lint.pydocstyle]
+convention = "numpy"
```
Comment (ahuang11) [inline template/pyproject.toml.jinja:126]: I think I prefer google docstrings.

EDIT: Wow I just realized Panel is using numpy docstyle so disregard this comment.

File: tests/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+"""Tests Module."""
```
Comment (ahuang11) [inline tests/__init__.py]:
```suggestion
"""Tests Module."""

```

File: template/tests/ui/__init__.py.jinja [hunk 5]
```diff
@@ -0,0 +1 @@
+"""UI Test Module."""
```
Comment (ahuang11) [inline template/tests/ui/__init__.py.jinja]:
```suggestion
"""UI Test Module."""

```

File: template/src/{{ project_slug_snake_case }}/main.py [hunk 6]
```diff
@@ -1 +1 @@
-import panel as pn
\ No newline at end of file
+"""Main module."""
```
Comment (ahuang11) [inline template/src/{{ project_slug_snake_case }}/main.py]:
```suggestion
"""Main module."""

```

File: template/tests/__init__.py.jinja [hunk 7]
```diff
@@ -0,0 +1 @@
+"""Tests Module."""
```
Comment (ahuang11) [inline template/tests/__init__.py.jinja]:
```suggestion
"""Tests Module."""

```

**Comment by ahuang11** [comment]
Thanks! I support these changes; I think just need to fix the linting, and if you're up for it, fixing the other CI (if not I can take a deeper look later and we can first merge this PR).

File: template/pyproject.toml.jinja (hunk 3 above)
Comment (MarcSkovMadsen) [reply template/pyproject.toml.jinja:126]: I do also prefer `google`. But I'm ok with aligning with rest of HoloViz which is numpy style.

**Comment by MarcSkovMadsen** [comment]
build-docs-via-github-actions

Users should select "Github Actions" to avoid also building docs via jekyll and sometimes publishing thoses.

**Comment by MarcSkovMadsen** [comment]
I looked through the failing tests. Its not obvious to me that they are related to this PR?

**Comment by ahuang11** [comment]
Correct it's unrelated. Will review this later and merge!

File: README.md [hunk 8]
```diff
@@ -1,59 +1,76 @@
...
+### Step 3 - Set Up GitHub Pages Docs
 
-### Step 2 - Setup GitHub Pages docs
+Enable GitHub Pages through **Settings > Pages** on the GitHub toolbar:
 
-Enable GitHub Pages through Settings > Pages on the GitHub toolbar:
+<img width="710" alt="image" src="https://private-user-images.githubusercontent.com/42288570/389284308-9bd345e5-7f3f-47c6-80e9-719063396f27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0NjE5NTAsIm5iZiI6MTczMjQ2MTY1MCwicGF0aCI6Ii80MjI4ODU3MC8zODkyODQzMDgtOWJkMzQ1ZTUtN2YzZi00N2M2LTgwZTktNzE5MDYzMzk2ZjI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI0VDE1MjA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2ZTZjM2ZlOGFjMjBjNWM2YmRkZjE4ZjNkMzMyZjczMDVkOGFiMWM3NTA1OTVmZTY4NzE3NTdmZDE1NTM4ODImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rj-hZ-dWRsQUgaE4FYd2H9VBdK7UyBC09zt2P8PAv_c">
```
Comment (ahuang11) [inline README.md]: This image is broken I think.

File: README.md [hunk 9]
```diff
@@ -1,59 +1,76 @@
...
 
-<img width="710" alt="image" src="https://github.com/user-attachments/assets/790f4a3e-31ca-42d7-b36e-27e794a504d1">
+On your GitHub repo's **About** section in the sidebar, be sure to add the link to your docs page:
 
-On your GitHub repo's sidebar's About section, be sure to add the link to your docs page and some tags!
+https://<github-user>.github.io/<panel-extension-name>/
```
Comment (ahuang11) [inline README.md]: I think there's a button to automatically use the GitHub page.
image

File: README.md [hunk 10]
```diff
@@ -1,59 +1,76 @@
...
-### Step 3 - Link to PyPi
+### Step 4 - Link to PyPI
 
-Head over to https://pypi.org/manage/account/publishing and fill out the form:
+Head over to [PyPI](https://pypi.org/manage/account/publishing) and fill out the form:
 
 <img width="797" alt="image" src="https://github.com/user-attachments/assets/69ea7626-a2df-4fbe-a1d4-d20eb5e0cdd7">
 
-#### Step 4 - Release with a tag
+### Step 4 - Release with a Tag
```
Comment (ahuang11) [inline README.md]:
```suggestion
### Step 5 - Release with a Tag
```

**Review by ahuang11 — APPROVED** [review]
Looks good to me! Just minor comments.

File: README.md (hunk 8 above)
Comment (MarcSkovMadsen) [reply README.md]: fixed by adding the image to the repository.

File: README.md (hunk 9 above)
Comment (MarcSkovMadsen) [reply README.md]: I'm not sure I understand. Could you make the change?

File: README.md [hunk 11]
```diff
@@ -1,59 +1,76 @@
...
+### Step 3 - Set Up GitHub Pages Docs
 
-### Step 2 - Setup GitHub Pages docs
+Enable GitHub Pages through **Settings > Pages** on the GitHub toolbar:
 
-Enable GitHub Pages through Settings > Pages on the GitHub toolbar:
+<img width="710" alt="image" src="assets/review-feedback.png">
 
-<img width="710" alt="image" src="https://github.com/user-attachments/assets/790f4a3e-31ca-42d7-b36e-27e794a504d1">
+On your GitHub repo's **About** section in the sidebar, be sure to add the link to your docs page:
```
Comment (ahuang11) [inline README.md]:
```suggestion
On your GitHub repo's **About** section in the sidebar, be sure to add the link to your docs page by toggling the checkbox:

![image](https://github.com/user-attachments/assets/69d70fee-ec96-47bb-87df-08f866b61f5f)
```

**Comment by ahuang11** [comment]
Thanks for helping me improve this significantly!

**Comment by MarcSkovMadsen** [comment]
Thanks for creating this amazing piece of art!

**Pull request merged**